manifests/config.pp

```diff
@@ -15,8 +15,13 @@
     mode    => $logrotate::rules_configdir_mode,
   }
 
-  if $manage_cron_daily {
-    logrotate::cron { 'daily': }
+  $cron_ensure = $manage_cron_daily ? {
```

Comment:
why was this part changed?

Reply:
For the same reason I explained below. 

When someone elects to not let the module manage the daily cron entry the entry should be removed from /etc/cron.daily by passing the appropriate ensure value through to logrotate::cron. 

With the code the way it was if someone originally ran with manage_cron_daily = true and then moved to manage_cron_daily = false the /etc/cron.daily entry is not cleaned up.